test/C/src/AfterZero.lf

```diff
@@ -5,6 +5,10 @@ target C {
     timeout: 3 sec
 }
 
+preamble {=
+    #include <stdio.h>
```

Comment:
Why are we doing this? I can come up with arguments for and against providing `stdio` through the runtime. What was the deciding factor that lead to this change?

Reply:
No deciding factor in particular. I decided after the name conflict with `windows.h` that I wanted to make the user-visible namespace as clean as reasonably possible. Namespace pollution seems like the sort of thing that can seem fine for a long time, but that turn out to cause weird, unexpected problems as you write larger, more complex programs, so I took an absolutist position rather than considering each `#include` on a case-by-case basis.

However, if we were to consider the `#include`s on a case-by-case basis, then `stdio.h` would be among the most likely things that you would allow into the user-visible namespace by default. It wouldn't be unreasonable to include it.